apps/server/lib/lexical/server/code_intelligence/completion/env.ex

```diff
@@ -112,21 +113,46 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Env do
     end
   end
 
+  @impl Environment
+  def in_context?(%__MODULE__{} = env, :struct_arguments) do
+    cursor_path = Ast.cursor_path(env.document, env.position)
+
+    Enum.any?(cursor_path, fn
+      # struct leading by current module: `%__MODULE__.Struct{|}`
+      # or leading by a module alias: `%Alias.Struct{|}`
+      # or just a struct: `%Struct{|}`
+      {:%, _, [{:__aliases__, _, _aliases} | _]} -> true
+      # current module struct: `%__MODULE__{|}`
+      {:%, _, [{:__MODULE__, _, _} | _]} -> true
+      _ -> false
+    end)
+  end
+
+  @impl Environment
+  def in_context?(%__MODULE__{} = env, :struct_field_value) do
```

Comment:
won't this return true for arguments and static keyword lists? 
I think you'll need to ensure you're in struct arguments too.